app/Http/Controllers/Littercoin/LittercoinController.php

```diff
@@ -24,16 +32,270 @@ public function getUsersLittercoin ()
     }
 
     /**
-     * Sign the transaction
-     *
-     * Submit the transaction
-     *
-     * Update the Littercoin as sent
+     * Get the amount Ada and littercoin in circulation and
+     * the littercoin script address and UTXO with the thread
+     * token in it.
+     */
+    public function getLittercoinInfo () {
+
+        $cmd = '(cd ../littercoin/;node get-lc-info.mjs) 2>> ../storage/logs/littercoin.log'; 
+        $response = exec($cmd);
+
+        return [
+            $response
+        ];
+    }
+
+
+    /**
+     * Build the littercoin mint transaction.
+     */
+    public function mintTx (Request $request) {
+
+        $destAddr = $request->input('destAddr');
+        $changeAddr = $request->input('changeAddr');
+        $utxos = $request->input('utxos');
+        $strUtxos=implode(",",$utxos);
+
+        $userId = Auth::user()->id;
+        $littercoinPaid = Auth::user()->littercoin_paid;
```

Comment:
I think we should calculate this from the littercoin table and update each littercoin as sent or not

Reply:
agreed, with the latest commits, we are now only reading and update the littercoin table